### ISSUE TITLE
WYSIWYG fields insert image does not respect the setting of WYSIWYG_IMAGE_URL_PATTERN

### ISSUE THREAD

**Issue by swicken-dotcms** [dotCMS/core#20642]
**Describe the bug**
When you insert an image into the WYSIWYG editor it does not respect the setting for WYSIWYG_IMAGE_URL_PATTERN and instead defaults to the contentAsset path every time.

Per our documentation this is how you configure the pattern: 
https://dotcms.com/docs/latest/the-wysiwyg-field#ImageURLFormat

Reproduced in 21.06 and 5.3.8.5 

Per ticket: https://dotcms.zendesk.com/agent/tickets/104528

**To Reproduce**
1. Set `WYSIWYG_IMAGE_URL_PATTERN={path}{name}` in `dotmarketing-config.properties`
2. Add an image to a WYSIWYG field
3. Switch to code, you will see the src on the image is the contentAsset path and not the path defined in settings.

**Expected behavior**
It should respect the path defined in settings.

**Additional context**
http://github.com/dotCMS/core/blob/0114793f4187af9eb1c98dcdf67bf68a80ecd4e4/dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/ResourceLink.java#L196-L196

In code it looks like the default path format is meant to be `{path}{name}?language_id={languageId}` so we are also not adding language IDs.

**Comment by jdotcms** [comment]
Here https://dotcms.com/docs/latest/the-wysiwyg-field

The default should be change from: 
```
WYSIWYG_IMAGE_URL_PATTERN={path}{name}?language_id={languageId}
```

to

```
WYSIWYG_IMAGE_URL_PATTERN=/dA/{shortyInode}/{name}
```

**Comment by rweiner** [comment]
Doc change has already been made to The Wysiwyg Field doc in auth. Just need to push to prod

**Comment by victoralfaro-dotcms** [comment]
Setting this path
```
WYSIWYG_IMAGE_URL_PATTERN={path}xxxxxxxxx{name}
```
WYSIWYG
![image](https://user-images.githubusercontent.com/64026161/127074559-cb60947c-2a70-40cf-b771-121f029f1fea.png)
Code
![image](https://user-images.githubusercontent.com/64026161/127074604-f0cb4694-859c-48e6-8057-a7cffc3f7337.png)

**Comment by bryanboza** [comment]
Fixed, tested on release-21.09 // Postgres // FF
